### PR TITLE
Support for feature/background_photo_uploads SDK branch

### PR DESCRIPTION
**Pull request by mattw-mega**
Support for the background photo uploads PR in the SDK (https://github.com/meganz/sdk/pull/1339)

Passes the Original Fingerprint through files sent via chat, in the same way the Fingerprint is sent.
Also adjusted `services_get_time_ms()` for Windows and MSVC, as it doesn't have `gettimeofday()`.

### CODE REVIEW

File: examples/megaclc/megaclc.cpp [hunk 1]
```diff
@@ -2119,6 +2119,89 @@ void exec_getfingerprint(ac::ACState& s)
...
+    string parallelcount;
+    bool tempmegaapi = extractflag("-tempmegaapi", s.words);
+    bool parallel = extractflagparam("-parallel", parallelcount, s.words);
+
+    if (!parallel)
+    {
+        parallelcount = "1";
+    }
+
+    vector<unique_ptr<thread>> ts;
```
Comment (sergiohs84) [inline examples/megaclc/megaclc.cpp]: @mattw-mega, building on Linux:
```error: ‘thread’ was not declared in this scope
vector<unique_ptr<thread>> ts;```

Comment (mattw-mega) [reply examples/megaclc/megaclc.cpp]: sorry about that- fixed now

**Comment by mattw-mega** [comment]
trigger compilation

**Comment by mattw-mega** [comment]
trigger compilation

**Comment by mattw-mega** [comment]
trigger compilation

**Pull request merged**